pyproject.toml

```diff
@@ -23,3 +34,20 @@ before-all = [
 ]
 # Skip musllinux builds for the moment
 skip = "*-musllinux_*"
+
+[tool.scikit-build]
+metadata.version.provider = "scikit_build_core.metadata.setuptools_scm"
+cmake.version = ">=3.11.0"
+sdist.include = ["src/fastjet/version.py"]
+wheel.install-dir = "fastjet"
+
+[tool.scikit-build.cmake.define]
+FASTJET_ENABLE_PYTHON = "ON"
+FASTJET_ENABLE_CGAL = "ON"
+FASTJET_PYTHON_PACKAGE_NAME = "_swig"
+FASTJET_CUSTOM_PYTHON_INSTALL = "./"
+CMAKE_BUILD_TYPE = "Release"
```

Comment:
You don't need this, it's done by scikit-build-core, unless you are building with a debug Python. And it's got it's own scikit-build-core setting.